src/elements/Checkbox.tsx

```diff
@@ -54,12 +61,21 @@ const Checkbox: ForwardRefRenderFunction<HTMLInputElement, CheckboxProps> = (
         { ...rest }
       />
 
-      { errorMessage && (
-        <Box mt={ 0.5 }>
-          <ErrorMessage>{ errorMessage }</ErrorMessage>
-        </Box>
-      ) }
-    </>
+      <Stack direction="column">
+        { typeof label === "string" ? (
+          <Typography
+            variant="subtitle1"
+            sx={ { color: theme.colors.white, fontSize: 12 } }
+          >
+            { label }
+          </Typography>
+        ) : (
+          label
+        ) }
```

Comment:
Is this just a general pattern to provide a default instead of typing the parameter in one direction? Is there reason to leave the `ConfirmContract.tsx` in a Typography wrapper?